### PR TITLE
OJ-2479: make persistent id optional

### PR DESCRIPTION
**Pull request by jkunle**
## Proposed changes

Make the Persistent Session Id optional

**Check Session was:**
image

**Now:**

image

### Why did it change

Core integration broke in staging due to the assumption that it was always present in the JAR request

- [OJ-2479](https://govukverify.atlassian.net/browse/OJ-2479)

[OJ-2479]: https://govukverify.atlassian.net/browse/OJ-2479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: integration-tests/tests/mocked/nino-issue-credential/MockConfigFile.json [hunk 1]
```diff
@@ -114,6 +125,42 @@
         }
       }
     },
+    "QuerySessionItemWithPersistentSessionId": {
```
Comment (Caitlin-cooling) [inline integration-tests/tests/mocked/nino-issue-credential/MockConfigFile.json:128]: This is called `QuerySessionItemWithPersistentSessionId` but it doesn't appear to have a persistent session ID in it?

File: integration-tests/tests/mocked/nino-issue-credential/nino-issue-credential-happy.test.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { HistoryEvent } from "@aws-sdk/client-sfn";
 import { SfnContainerHelper } from "./sfn-container-helper";
 
-jest.setTimeout(30_000);
+jest.setTimeout(60_000);
```
Comment (Caitlin-cooling) [inline integration-tests/tests/mocked/nino-issue-credential/nino-issue-credential-happy.test.ts:4]: How come we've had to double the time out?

File: integration-tests/tests/mocked/nino-issue-credential/MockConfigFile.json (hunk 1 above)
Comment (jkunle) [reply integration-tests/tests/mocked/nino-issue-credential/MockConfigFile.json:128]: This was a typo, should have been `QuerySessionItemWithoutPersistentSessionId`

File: integration-tests/tests/mocked/nino-issue-credential/nino-issue-credential-happy.test.ts (hunk 2 above)
Comment (jkunle) [reply integration-tests/tests/mocked/nino-issue-credential/nino-issue-credential-happy.test.ts:4]: I think these test started with 5_000 we've been doubling this, whenever we encounter an issue where we run these test locally and they pass, but then fail when github action runs them. Mainly it is something to do with do initialisation we have this ticket https://govukverify.atlassian.net/browse/OJ-2188

Comment (Caitlin-cooling) [reply integration-tests/tests/mocked/nino-issue-credential/nino-issue-credential-happy.test.ts:4]: Cool great to see the investigation captured in a ticket will resolve the comment 👍

**Pull request merged**